sencha-workspace/SlateDemonstrationsTeacher/app/controller/Dashboard.js

```diff
@@ -2,8 +2,8 @@
  * TODO:
  * - move rendering responsibilities to the view?
  */
-Ext.define('SlateDemonstrationsTeacher.view.DashboardController', {
-    extend: 'Ext.app.ViewController',
+Ext.define('SlateDemonstrationsTeacher.controller.Dashboard', {
+    extend: 'Ext.app.Controller',
     alias: 'controller.slate-demonstrations-teacher-dashboard',
```

Comment:
@aliyeysides non-viewcontroller controllers shouldn't have any alias defined as they are singletons. Aliases are a tool for creating instances of a certain class via a registering it to a short name, but singletons don't get instantiated so there's no value to polluting the `controller.*` alias namespace